src/igvfd/search_views.py

```diff
@@ -360,6 +362,68 @@ def missing_matrix(context, request):
     return fr.render()
 
 
+@view_config(route_name='tissue-homo-sapiens', request_method='GET', permission='search')
+def tissue_homo_sapiens(context, request):
+    fr = FieldedResponse(
+        _meta={
+            'params_parser': ParamsParser(request)
+        },
+        response_fields=[
+            TitleResponseField(
+                title='Homo Sapiens Tissue Summary'
+            ),
+            TypeResponseField(
+                at_type=['TissueSummaryHomoSapiens']
+            ),
+            IDResponseField(),
+            SearchBaseResponseField(),
+            ContextResponseField(),
+            MissingMatrixWithFacetsResponseField(
+                default_item_types=DEFAULT_ITEM_TYPES,
+                matrix_definition_name='tissue_homo_sapiens',
```

Comment:
I think remove this and add `config=['whatever-you-name-your-search-config']` (I think string instead of list also works), so the endpoint uses your search config with the matrix definition.